packages/generator/ni_measurement_plugin_client_generator/template.py

```diff
@@ -0,0 +1,60 @@
+"""Utilizes command line args to create a measurement client using template files."""
+
+from typing import Sequence, Tuple
+
+import click
+from ni_measurement_plugin_sdk_service.discovery import DiscoveryClient
+from ni_measurement_plugin_sdk_service.measurement.info import ServiceInfo
+
+
+_V2_MEASUREMENT_SERVICE_INTERFACE = "ni.measurementlink.measurement.v2.MeasurementService"
+
+
+def _get_module_name_and_service_class(
```

Comment:
Would it make sense to split this method to be more granular? Where one method will take care of displaying the registered services and another method to get the input index?

By this change, I guess we can call both methods in the `create_client` itself.